exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -338,3 +338,11 @@ drill.exec: {
 drill.jdbc: {
   batch_queue_throttling_threshold: 100
 }
+javax.net.ssl.keyStore = ""
+ssl.keyStorePath = ${?javax.net.ssl.keyStore}
+javax.net.ssl.keyStorePassword = ""
+ssl.keyStorePassword = ${?javax.net.ssl.keyStorePassword}
+javax.net.ssl.trustStore = ""
+ssl.trustStorePath = ${?javax.net.ssl.trustStore}
+javax.net.ssl.trustStorePassword = ""
+ssl.trustStorePassword =  ${?javax.net.ssl.trustStorePassword}
```

Comment:
Add a comment about the `javax` versions being legacy and deprecated.

```
javax.net.ssl: {
  keyStorePath = ...
  ...
}
drill.exec.ssl {
  keyStorePath = "",
  ...
}
```

Better, move the `drill.exec` stuff under the existing `drill.exec` section.